langchain/vectorstores/deeplake.py

```diff
@@ -171,11 +172,19 @@ def add_texts(
             texts (Iterable[str]): Texts to add to the vectorstore.
             metadatas (Optional[List[dict]], optional): Optional list of metadatas.
             ids (Optional[List[str]], optional): Optional list of IDs.
-            **kwargs: other optional keyword arguments.
+            embedding_function (Optional[Embeddings], optional): Embedding function
+                to use to convert the text into embeddings.
+            **kwargs (Any): Additional keyword arguments
```

Comment:
can we include in docstring that these shouldn't actually be passed in, and an error will be raised if they are